### PR TITLE
docs: add links to examples

### PR DESCRIPTION
**Pull request by clemcode**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This PR adds links to examples in the relevant sections and extracts the `createTitle` function to reuse it.

### CODE REVIEW

File: docs/components/templates/ReadMore.vue [hunk 1]
```diff
@@ -22,8 +22,7 @@ export default defineComponent({
   computed: {
     computedTitle () {
       // Guess title from link!
-      return this.title || this.link.split('/')
-        .filter(Boolean).map(part => splitByCase(part).map(p => upperFirst(p)).join(' ')).join(' > ')
+      return createTitle(this.title, this.link)
```
Comment (danielroe) [inline docs/components/templates/ReadMore.vue:25]: Would be nice if `Api` rendered as `API` somehow...

Comment (Atinux) [reply docs/components/templates/ReadMore.vue:25]: I guess we can make an exception in this code, like `.replace('Api', 'API')`

**Comment by Atinux** [comment]
Another note, we can use a single `:` in order to use a component without slots:

```
::LinkExample{link="/examples/app/teleport"}
::
```

Becomes:

```
:LinkExample{link="/examples/app/teleport"}
```

File: docs/utils/createTitle.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { splitByCase, upperFirst } from 'scule'
+
+export default (title, link) => {
+  return title || link.split('/').filter(Boolean).map(part => splitByCase(part).map(p => upperFirst(p)).join(' ')).join(' > ')
```
Comment (Atinux) [inline docs/utils/createTitle.js:4]:
```suggestion
  return title || link.split('/').filter(Boolean).map(part => splitByCase(part).map(p => upperFirst(p)).join(' ')).join(' > ').replace('Api ', 'API')
```

To check locally 😄

File: docs/components/templates/ReadMore.vue (hunk 1 above)
Comment (clemcode) [reply docs/components/templates/ReadMore.vue:25]: Good point, updated in last commit!

File: docs/utils/createTitle.js (hunk 2 above)
Comment (clemcode) [reply docs/utils/createTitle.js]: Works 🙂

**Review by danielroe — APPROVED** [review]

**Review by Atinux — APPROVED** [review]

**Pull request merged**